src/nvim/main.c

```diff
@@ -341,6 +341,17 @@ int main(int argc, char **argv)
   source_startup_scripts(&params);
 
   /*
+   * If using the runtime (-u is not NONE),
+   * enable syntax and filetype plugins
+   */
+  if (params.use_vimrc != NULL && strcmp(params.use_vimrc, "NONE") != 0) {
```

Comment:
Is that necessary? See line 1668 and following:
~~~ c
  if (parmp->use_vimrc != NULL) {
    if (strcmp(parmp->use_vimrc, "NONE") == 0
        || strcmp(parmp->use_vimrc, "NORC") == 0) {
      if (parmp->use_vimrc[2] == 'N')
        p_lpl = false;  // don't load plugins either
    } else {
~~~